aries-site/src/pages/templates/content-layouts.mdx

```diff
@@ -32,39 +71,89 @@ Specifying layout content in this way will also ensure that it is interepreted a
+which provides Grid-based layout templates for each [responsive breakpoint](#breakpoints) and [page layout kind](/templates/page-layouts#page-kinds).
+
+### Container-Driven Layouts
+Container-driven layouts drive the layout from the outside-in. They begin by defining 
+regions in which the content will live, whether those regions are [fixed, flexible, or fluid](/components/grid#background), 
+and how those regions are [prioritized](#prioritizing-content-needs) at each responsive breakpoint.
+
+Container-driven layouts are useful for defining larger layouts, such as at the page level, 
+but can be useful for defining smaller sub-regions as well. The best use of available screen 
+real-estate can be achieved by combining a container-driven page layout with things like Cards
+which have been built to have adaptive layouts themselves.
+
+The following example takes a container-driven approach, implementing a an app-catalog 
+consisting of a filters region and a results region.
+
+Key aspects to the approach are:
+- At larger breakpoints, the content layout will be a two-column layout; a single column 
+at smaller breakpoints.
+- The filters region will be fixed-width when in a two-column layout and [flex-width](/components/grid#flexible-columns) 
+when in a single column. The results region always uses a [fluid grid](/components/grid#fluid-grids).
+- The filters region are prioritized over the results region because the filters are needed to control 
+the results.
+
+<Example
+  showResponsiveControls={['fullScreen']}
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/ContainerDrivenLayout.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/PageHeader/PageHeader.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/AppResults.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/FilterPanel.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/data/mockData/applications.json",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/constants.js",
+    ]}
+  github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  screenContainer={{ scale: 0.75 }}
+>
+  <ContainerDrivenLayout />
```

Comment:
Wondering more broadly about patterns like this -- do we ever make the filters panel sticky as the user scrolls down so they can always be accessed?

Reply:
Yes, I envision this like an e-commerce experience where the filters/controls are fixed. In the example when I opened the deploy, it scrolled off screen when I scrolled down the cards.